src/main/java/com/example/event_manager/service/EventServiceImpl.java

```diff
@@ -152,6 +147,11 @@ public AllEventsForm searchByNamePlaceTopic(final String s, final Pageable pagin
         .build();
   }
 
+  @Override
+  public BillingRaportSchema generateBillingRaportSchemaForEvent(Long id) {
```

Comment:
remove it if unused
return NotImplementedException exception if implementation will be provided in future

Reply:
+ final param